HP.CloudApi.Client/HeimdallApiClient.cs

```diff
@@ -12,22 +12,17 @@ namespace HeimdallPower;
 /// <summary>
 /// A client that lets you consume the Heimdall Cloud API
```

Comment:
```suggestion
/// A client that lets you consume the Heimdall Power API
```

<img width="570" height="570" alt="image" src="https://github.com/user-attachments/assets/79f3dba1-bf3e-4d58-acdb-1f37d97a44cd" />